### PR TITLE
Ha mf global nav update #4271 [ staging ]

### PR DESCRIPTION
**Pull request by matt-vc4a**
# Description
Small tweaks to globally experienced styling issues over the global and context headers

## Type of change
Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] An example/test case has been set up, which you can found here: https://staging-vc4a.com/... (production PR only)

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: scss/00-core/_icon.scss [hunk 1]
```diff
@@ -1,6 +1,8 @@
+$icon-size: $spacing-06;
```
Comment (olufemioye) [inline scss/00-core/_icon.scss:1]: Icon size should not use spacing variables

Comment (matt-vc4a) [reply scss/00-core/_icon.scss]: Nice. Switched to rem value.

(still needs to be available as a variable for an offset calculation in the context-header)

[Commit](https://github.com/vc4a-dev/vc4a-styles/pull/98/commits/071e64ae10bac153059ac5828a03d7d6aac79703)

Comment (matt-vc4a) [reply scss/00-core/_icon.scss]: Nice. Switched to rem value.

(still needs to be available as a variable for an offset calculation in the context-header)

[Commit](https://github.com/vc4a-dev/vc4a-styles/pull/98/commits/071e64ae10bac153059ac5828a03d7d6aac79703)

@olufemioye

Comment (matt-vc4a) [reply scss/00-core/_icon.scss]: Actually... maybe the variable should be in variables.scss

But I didn't want to set a precedent for moving component-specific variables. Let me know if you want me to move it @olufemioye

**Review by olufemioye — APPROVED** [review]

**Pull request merged**